python_ta/reporters/node_printers.py

```diff
@@ -132,6 +132,13 @@ def render_missing_space_in_doctest(msg, _node, source_lines=None):
     yield from render_context(line + 1, line + 3, source_lines)
 
 
+def get_col(msg):
+    """Returns the column number of the character containing the error"""
```

Comment:
In general, use the imperative verb tense for docstrings (so "Return" rather than "Returns")